reference-architecture/3.9/playbooks/roles/aws/tasks/iam.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Fetch IAM idenity
+  set_fact:
+    iam_identity: "{{ lookup('iam_identity') | replace (',', '') }}"
+
+- name: Create IAM user for cloudprovider_kind
+  iam:
+    iam_type: user
+    name: "{{ clusterid }}-admin"
+    state: present
+    access_key_state: create
+  register: cpkuser
+
+- name: "Pause for 15s to allow for IAM to refresh"
+  pause:
+    seconds: 15
+  when: cpkuser.changed
+
+- name: Land IAM cloudprovider_kind access key into file
+  blockinfile:
+    path: "~/.ssh/config-{{ clusterid }}-cpkuser_access_key"
+    create: yes
+    marker: "<!-- {mark} ANSIBLE MANAGED BLOCK -->"
+    content: |
+      openshift_cloudprovider_aws_access_key={{ cpkuser.user_meta.access_keys[0].access_key_id }}
```

Comment:
have you thought about just attaching a role to the instances?

Reply:
Yes.  If the host gets popped an unpriv'd user could wreck havoc with the aws api.  We need to discuss more before jumping into an ec2 role

Reply:
:+1: